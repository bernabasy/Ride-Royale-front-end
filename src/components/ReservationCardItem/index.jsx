import React from 'react';
import style from './style.module.css';

const ReservationCardItem = () => (
  <div className="col-md-6 my-2">
    <div className="card p-2">
      <div className="row g-0">
        <div className="col-md-4">
          <img className={`img-fluid rounded ${style['card-img']}`} src="https://images.pexels.com/photos/3802510/pexels-photo-3802510.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="luxury car" />
        </div>
        <div className="col-md-8">
          <div className="card-body">
            <h5 className="card-title">Card Model: Model</h5>
            <p className="card-text">Reservation date: </p>
            <p className="card-text">Reservation city: </p>
            <p className="card-text">Driver&apos;s Name: </p>
            <p className="card-text">Charge: </p>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default ReservationCardItem;
