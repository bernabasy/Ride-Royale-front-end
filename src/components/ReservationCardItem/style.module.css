.card-wrapper {
  min-height: 120px;
}

.title {
  color: #96bf01;
  margin-left: 0.8em;
}