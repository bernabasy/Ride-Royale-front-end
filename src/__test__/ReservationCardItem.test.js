import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import ReservationCardItem from '../components/ReservationCardItem';

describe('ReservationCardItem tests', () => {
    it('renders correctly', () => {
      const tree = render(
        <BrowserRouter>
            <ReservationCardItem />
          </BrowserRouter>
      );
      expect(tree).toMatchSnapshot();
    });
  });