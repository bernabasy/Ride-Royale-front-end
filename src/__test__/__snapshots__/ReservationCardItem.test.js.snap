// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCardItem tests renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="col-md-6 my-2"
      >
        <div
          class="card p-2 shadow-sm card-wrapper"
        >
          <div
            class="row g-0"
          >
            <div
              class="col-md-4"
            >
              <img
                alt="luxury car"
                class="img-fluid rounded card-img"
                src="https://images.pexels.com/photos/3802510/pexels-photo-3802510.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
              />
            </div>
            <div
              class="col-md-8"
            >
              <div
                class="card-body"
              >
                <h5
                  class="card-title"
                >
                  Card Model: Model
                </h5>
                <p
                  class="fs-6 m-0"
                >
                  Reservation date: 
                </p>
                <p
                  class="fs-6 m-0"
                >
                  Reservation city: 
                </p>
                <p
                  class="fs-6 m-0"
                >
                  Driver's Name: 
                </p>
                <p
                  class="fs-6 m-0"
                >
                  Charge: 
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="col-md-6 my-2"
    >
      <div
        class="card p-2 shadow-sm card-wrapper"
      >
        <div
          class="row g-0"
        >
          <div
            class="col-md-4"
          >
            <img
              alt="luxury car"
              class="img-fluid rounded card-img"
              src="https://images.pexels.com/photos/3802510/pexels-photo-3802510.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
            />
          </div>
          <div
            class="col-md-8"
          >
            <div
              class="card-body"
            >
              <h5
                class="card-title"
              >
                Card Model: Model
              </h5>
              <p
                class="fs-6 m-0"
              >
                Reservation date: 
              </p>
              <p
                class="fs-6 m-0"
              >
                Reservation city: 
              </p>
              <p
                class="fs-6 m-0"
              >
                Driver's Name: 
              </p>
              <p
                class="fs-6 m-0"
              >
                Charge: 
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
